lib/utils/utils.js

```diff
@@ -31,7 +31,7 @@ const readJSON = (sink, path) => {
 module.exports.readJSON = readJSON;
 
 const writeJSON = (sink, path, obj, contentType) => {
-    return new Promise((resolve, reject) => {
+    return new Promise(async (resolve, reject) => {
```

Comment:
link will probably moan about using an async function in a new Promise call. Might have to ignore